plugin/src/main/java/com/denizenscript/denizen/events/entity/AreaEffectCloudApplyScriptEvent.java

```diff
@@ -0,0 +1,58 @@
+package com.denizenscript.denizen.events.entity;
+
+import com.denizenscript.denizen.events.BukkitScriptEvent;
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ListTag;
+import org.bukkit.entity.Entity;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.entity.AreaEffectCloudApplyEvent;
+
+public class AreaEffectCloudApplyScriptEvent extends BukkitScriptEvent implements Listener {
+
+    // <--[event]
+    // @Events
+    // area cloud applies effects

```

Comment:
Should probably mention it's an effect cloud somewhere in the name, this sounds like it's a normal cloud doing something - `area effect cloud applies effects`, `effect cloud applies effects`, `area effect cloud applies`... can discuss names on Discord if you're not sure